File: src/main/java/ru/diasoft/micro/dofacade/mock/ProductApplicationRes.java
package ru.diasoft.micro.dofacade.mock;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductApplicationRes {

    @JsonProperty("CardNumber")
    String cardNumber = "1111 1111 1111 1111";

    @JsonProperty("DOCTYPENAME")
    String typeName = "Issue virtual card";

    @JsonProperty("STATE")
    String cardStatus = "Checks";

    @JsonProperty("DOCTYPESYSNAME")
    String productType = "virtualCardApp";

    @JsonProperty("Name")
    String cardHolder = "Demo User";

}

File: src/main/java/ru/diasoft/micro/dofacade/json/product/ProductApplicationRes.kt
package ru.diasoft.micro.dofacade.json.product

import com.fasterxml.jackson.annotation.JsonProperty


data class ProductApplicationRes(

    @JsonProperty("CardNumber")
    val cardNumber: String?,

    @JsonProperty("DOCTYPENAME")
    val typeName: String?,

    @JsonProperty("STATE")
    val cardStatus: String,

    @JsonProperty("DOCTYPESYSNAME")
    val productType: String,

    @JsonProperty("Name")
    val cardHolder: String
)